### ISSUE TITLE
Double byte character spacing

### ISSUE THREAD

**Issue by asins** [HappenApps/Quiver#649]
In  Markdown cell，My code is：

  ```sh
  git checkout -b branchName origin/master # 基于远程创建并进入分支
  git push -u origin branchName # 与远程分支绑定关系方便以后提交
  ykpm build
  git commit -am "本次提交说明"
  git push
  ```

The preview is：
![image](https://cloud.githubusercontent.com/assets/898354/17093503/3a0b302e-527c-11e6-8fb3-fc7d01a0a125.png)

Double byte character spacing is too large. I want the same effect as in the code.

**Comment by ylian** [comment]
Are you running the latest version (3.0.3)? I believe this is already fixed.

**Comment by asins** [comment]
Yes，is fixed of V3.0.3, thanks.
